packages/airnode-examples/integrations/failing-example/request-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { encode } from '@api3/airnode-abi';
+import { cliPrint, getDeployedContract, readIntegrationInfo } from '../../src';
+
+const coinLabel = 'Ethereum';
+const coinId = coinLabel.toLowerCase();
+
+export const getEncodedParameters = () => {
+  return encode([{ name: 'coinId', type: 'string32', value: coinId }]);
+};
+
+export const printResponse = async (requestId: string) => {
```

Comment:
This function is not needed (will never be called since the request fails) but I am OK with it staying